corehq/apps/es/es_query.py

```diff
@@ -512,11 +506,10 @@ def doc_ids(self):
     @property
     def hits(self):
         """Return the docs from the response."""
-        raw_hits = self.raw_hits
-        if not raw_hits and self.query.uses_aggregations() and self.query._size == 0:
-            raise ESError("no hits, did you forget about no_hits_with_aggs?")
+        if self.query.uses_aggregations():
+            raise ESError("We exclude hits for aggregation queries")
```

Comment:
This reduces the snark factor of this file by 10. 👍🏽

Reply:
Not to mention that `no_hits_with_aggs` isn't a real flag/parameter!

Reply:
I think you may have just forgotten about it.

Reply:
:unamused: